### ISSUE TITLE
Visiting /experimental gives empty blank page with Firefox

### ISSUE THREAD

**Issue by TNW9imKLC3fv** [vector-im/riot-web#8082]
- **Browser**: Firefox 43,0.1 (one of the last good versions before Mozilla went crazy and made changes that broke a lot of important addons like Self Destructing Cookies with very little time for warning and no consent from users) with NoScript addon (allowing JavaScript from riot.im)
- **OS**: Windows 7 64-bit

### Description

Visiting the /experimental address gives nothing but a white page, even after allowing JavaScript from riot.im with NoScript (Discord works fine with the older browser)

### Steps to reproduce

- Visiting the /experimental address with Firefox 43.0.1 with NoScript, whilst riot.im JavaScript is set to allow

Blank page instead of any content shown.

Log: sent/not sent?    N/A, page has no no script content whatsoever, not even errors

**Comment by ara4n** [comment]
what does the javascript console say?

**Comment by jryans** [comment]
(As an aside, I would urge you to use an up to date browser, as Firefox 43 is now quite an old version, and there are literally hundreds of known vulnerabilities that could potentially affect you.)

It appears that somehow, a single module `SdkConfig` is not being compiled down to ES5, so it still has the keyword `class` from ES6 / ES2015 classes, which were first supported in Firefox 45.

**Comment by jryans** [comment]
It appears this is a regression because of the following line:

https://github.com/vector-im/riot-web/blob/710e295b44c78d72bc96011fe91f72148e0e9868/src/vector/rageshakesetup.js#L29

which [was added](https://github.com/vector-im/riot-web/commit/710e295b44c78d72bc96011fe91f72148e0e9868#diff-2c70402eb5fe094f3542fa7df53612b3R29) as part of v0.17.8. Since it uses `matrix-react-sdk/src` instead of `lib`, the untransformed original source is used, which includes the `class` keyword.

**Comment by jryans** [comment]
This should now be fixed on [experimental](https://github.com/vector-im/riot-web/pull/8109) and [develop](https://github.com/vector-im/riot-web/pull/8110). Please try experimental again to confirm it's fixed on your end as well.

**Comment by TNW9imKLC3fv** [comment]
Registration gets as far as "This Home Server would like to make sure you are not a robot" stage, but there is nothing beneath those words other than a language selector and page footer, no errors shown on page (scripts from matrix.org and riot.im allowed). https://stackoverflow.com/questions/2170484/shouldnt-we-use-noscript-element/10480048#10480048 might be relevant

Looking at NoScript I saw that it had blocked scripts from google.com without permission (because of the dubious though not-yet-illegal tracking without consent they do including invisible 1x1 gifs / web beacons), so I temporarily allowed google.com and refreshed the page but instead it took me to the main chat page as an anonymous account instead of continuing the registration (when I refreshed I didn't get an option to resubmit form data)

1. I manually went back to the registration page

2. I typed in the username and password twice again

3. Click register

4. Blank page for a second time, same issue, except this time it's gstatic.com - I refresh and again it loses all registration process without any option on refresh to resend the form data

5. Have to manually revist the registration page again

6. Have to refill the form again

7. Works, but might put some people off from keeping trying (needed to fill in registration form 3 times)
